packages/core/test/integration/query-interface/schemas.test.ts

```diff
@@ -32,6 +31,68 @@ describe('QueryInterface#{create,drop,list}Schema', () => {
     expect(postCreationSchemas).to.include(testSchema, 'createSchema did not create testSchema');
   });
 
+  if (dialect.supports.createSchema.authorization) {
```

Comment:
For these new tests; do we want to validate in some way that the options are actually used? Or is relying on the unit tests for this enough?

Reply:
Sure, I've updated the test now.
With db2 you cannot create users using SQL statements therefore I left it to use `CURRENT_USER` as this usually defaults to the logged in user.

Reply:
Thanks for this! I was also talking about the other options, could you add additional checks for those as well?

Reply:
Sure, added now